crypto/s2n_certificate.c

```diff
@@ -763,38 +763,38 @@ static int s2n_parse_x509_extension(struct s2n_cert *cert, const uint8_t *oid_fi
                 */
                 *critical = X509_EXTENSION_get_critical(x509_ext);
             }
-            *oid_value_out_len = len;
+            *ext_value_len = len;
             return S2N_SUCCESS;
         }
     }
 
     POSIX_BAIL(S2N_ERR_X509_EXTENSION_VALUE_NOT_FOUND);
 }
 
-int s2n_get_x509_extension_oid_value_length(struct s2n_cert *cert, const uint8_t *oid_field_in,
-                                            const uint32_t oid_field_in_len, uint32_t *oid_value_len)
+int s2n_cert_get_x509_extension_value_length(struct s2n_cert *cert, const uint8_t *oid,
+                                            const uint32_t oid_len, uint32_t *ext_value_len)
```

Comment:
Is `oid_len` used by anything anymore? We're now using `OBJ_txt2obj` which takes in a null terminated c string.